app/public/ai/index.html

```diff
@@ -22,27 +22,27 @@ <h1 id="role-text">You are the AI</h1>
                 <div>Pings</div>
                 <button
                   id="fire"
-                  class="button"
+                  class="ai-button"
                   onclick="AI.switchThreatType('fire')">
                   <img src="../images/ai/fire.png" alt="fire" class="button-icon" />
                 </button>
                 <button
                   id="breach"
-                  class="button"
+                  class="ai-button selected"
                   onclick="AI.switchThreatType('breach')">
                   <img src="../images/ai/breach.png" alt="breach" class="button-icon" />
                 </button>
                 <button
                   id="invader"
-                  class="button"
+                  class="ai-button"
                   onclick="AI.switchThreatType('invader')">
                   <img src="../images/ai/invader.png" alt="invader" class="button-icon" />
                 </button>
             </div>
             <div class="tool-group">
                 <div>Manage Threats</div>
-                <button class="button"> Assist</button>
-                <button class="button"> Sabotage </button>
+                <button class="ai-button"> Assist</button>
+                <button class="ai-button"> Sabotage </button>
```

Comment:
These assist/sabotage buttons still have the class `ai-button`, meaning that they'll also be given the `unselected` class when swapping ping type. I think we should make a new class to differentiate these sets of buttons, like `ai-ping-button` and `ai-manage-button`

Reply:
i did think about that, but i don't think it will be an issue if the ai page's buttons are intended to look similar to the human page's buttons. currently, all of the buttons on the human's page have the class `button` and the boost and inhibit buttons are actually given the `unselected` class as well, but there is no visible difference since those actions are not meant to stay selected.

if you think that you might change how the buttons are styled in the future, i could still go ahead and make that change (should we do the same for the human page as well?), so just let me know if i should change the class names of the ai action buttons or the human buttons or both